tests/test_service.py

```diff
@@ -180,3 +180,24 @@ def test_invalid_operation(self):
         created_wishlist.save()
         resp = self.app.get('/wishlists/%s' % created_wishlist.id)
         self.assertEqual(resp.status_code, status.HTTP_405_METHOD_NOT_ALLOWED)
+
+    def test_add_product_to_wishlist(self):
+        """ Test adding a product to a wishlist"""
+
+        # test_wishlist_product = WishlistProduct(wishlist_id=123, product_id=2)
+        test_wishlist = Wishlist(id=123, name='test', customer_id=1)
```

Comment:
Here I don't think you need to add `id=123` since the primary key is generated by database, not from input.

Reply:
I think you are right, let's remove it.